test/escrow-tests.js

```diff
@@ -4,22 +4,24 @@ const etherlime = require('etherlime');
 const utils = require('./util');
 const Token = require('./../build/Token.json');
 const ECTools = require('./../build/ECTools.json');
-const EscrowContract = require('./../build/Escrow_V2.json');
+const EscrowContract = require('./../build/Escrow_V3.json');
 
-// TODO More tests must be implemented for the full coverage of the functionality and testing corner cases
+const GAS_PRICE = 20000000000; // 20 gwei
+const GAS_LIMIT = 200000;
+
+const EXCESS_GAS_REFUND_UPPER_LIMIT = 3500; // The upper limit of exceeding gas that can be refunded and we will count the refund successfull
+const NUMBER_OF_TRANSACTIONS = 500; // Number of transactions that will be perfomed for the refund test
 
 describe('Escrow Contract', function () {
     this.timeout(5000);
-    let signer = accounts[3];
+    let dAppSigner = accounts[3];
```

Comment:
dAppSigner is the predefined wallet that is given from `ganache`.  I have named it dAppSigner because we have the term `signer` for the wallet that is signing the authorisation signatures. 

Unfortunately after etherlime 1.1.0 the account wallets are not reached with `account.wallet` but `account.signer` which is kind of confusing.

Good news is that every-time we use `dAppSigner` we are doing `dAppSigner.signer` so I can change the variable to `accounts[3].signer`